generator/OpaqueGen.cs

```diff
@@ -159,6 +159,18 @@ public override void Generate (GenerationInfo gen_info)
 			}
 
 			if (finalizer_needed) {
+				// Generate Dispose override.
+				sw.WriteLine ("\t\tpublic override void Dispose ()");
+				sw.WriteLine ("\t\t{");
+				sw.WriteLine ("\t\t\tif (Owned)");
+				if (dispose != null)
+					sw.WriteLine ("\t\t\t\t{0} (Raw);", dispose.CName);
+				else if (unref != null)
+					sw.WriteLine ("\t\t\t\t{0} (Raw);", unref.CName);
+				sw.WriteLine ("\t\t\tbase.Dispose();");
+				sw.WriteLine ("\t\t}");

```

Comment:
https://github.com/mono/gtk-sharp/blob/gtk-sharp-2-12-branch/glib/Opaque.cs#L75-L85

Is this necessary?